docs/requirements.txt

```diff
@@ -1,9 +1,24 @@
-# Direct dependencies
-mkdocs>=1.3.0
-Pygments>=2.12
-markdown>=3.3.6
-pymdown-extensions>=9.4
-mkdocs-material-extensions>=1.0.3
-mkdocs-git-revision-date-localized-plugin>=0.7.3
-mkdocs-material>=8.3.8
-mdx-truly-sane-lists>=1.2
+click==8.0.4
```

Comment:
We could enable dependable for this file if we care about automatic updates for our docs deps.

Reply:
Good idea. I'll add this.